src/kernels/variables/kernelVariables.ts

```diff
@@ -335,7 +336,7 @@ export class KernelVariables implements IJupyterVariables {
             // Should be a text/plain inside of it (at least IPython does this)
             if (output && output.hasOwnProperty('text/plain')) {
                 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-                const text = (output as any)['text/plain'].toString() as string;
+                const text = stripAnsi((output as any)['text/plain'].toString() as string);
```

Comment:
do we still need `as string` when we use `.toString()`

Reply:
No, we don't even need the `any`